### PR TITLE
feat: show neighborhoods on map & heatmap using api data

### LINKED ISSUE #16: bug: neighborhood and school content in the query field is not available
## Bug Definition
neighborhood and school content in the query field is not available

**discord username: alicangunduz#0001**

## Bug environment
https://musahit-harita-frontend.vercel.app/

## Describe how you are producing the bug step by step

1. Go to https://musahit-harita-frontend.vercel.app/
2. Click "Okul Sorgula"
3. Select province and district
4. No neighborhoods and no schools
5. Bug appears

## Expected Behaviour
Visibility of neighborhoods and schools

## Screen shots
image

### LINKED ISSUE #23: feat: ❗ Finalize the pins and heat map functionality per neighbourhood view
** discord username: @Mustafa Bolat#4560 **

### PR DESCRIPTION
**Pull request by usirin**
Closes #23 
Closes #16

### CODE REVIEW

**Comment by eraygundogmus** [comment]
şiir gibi olmuş, hangi idede yazdın? ben de onu kullanayım, belki yetenek bulaşır hehe

File: src/components/Map/Content.tsx [hunk 1]
```diff
@@ -16,12 +17,29 @@ import { MapClusterStyle } from "@/components/Map/Cluster/ClusterStyle";
...
+const locations = getAllNeighborhoods().map(
+  (hood): ChannelData => ({
+    properties: {
+      name: hood.name,
+      description: null,
+    },
+    intensity: DEFAULT_IMPORTANCY,
+    location: {
+      lat: hood.lng,
+      lng: hood.lat,
```
Comment (usirin) [inline src/components/Map/Content.tsx]: need to fix this on data level.

**Comment by puskuruk** [comment]
> şiir gibi olmuş, hangi idede yazdın? ben de onu kullanayım, belki yetenek bulaşır hehe

10x developerlar ide kullanmaz. Onlar sadece notepad/Notes kullanir.

File: src/components/Map/LayerControl.tsx [hunk 2]
```diff
@@ -19,12 +18,14 @@ type Props = {
...
 
+const getIntensity = (data: ChannelData): number => data.intensity;
+
 const useHeatmapPoints = (locations: ChannelData[]) => {
   return useMemo(
     () =>
       locations.map(
         (item) =>
-          [item.location.lng, item.location.lat, DEFAULT_IMPORTANCY] as Point
+          [item.location.lng, item.location.lat, getIntensity(item)] as Point
```
Comment (puskuruk) [inline src/components/Map/LayerControl.tsx:28]: Why we didn't get intensity directly as `item.intensity`?

Comment (usirin) [reply src/components/Map/LayerControl.tsx:28]: good q, i started with having a helper idea, then kept it that way. you're right we can just use `item.intensity` here.

**Review by cansirin — APPROVED** [review]
Perfection 💯

File: src/components/Map/LayerControl.tsx (hunk 2 above)
Comment (usirin) [reply src/components/Map/LayerControl.tsx:28]: i'll do this in a follow up pr

**Pull request merged**